### DISCUSSION TITLE
Translations for proof, remark, solution

### DISCUSSION THREAD

**Discussion by mchlbckr** [quarto-dev/quarto-cli#560] [Feature Requests]
Hi, 
the available translations for e.g. figure, table or theorem titles in the respective `_language.yml` files (e.g. `_language-de.yml` for German) do not contain any translations for the `proof`, `remark` and `solution` environments (see [documentation](https://quarto.org/docs/authoring/cross-references.html#theorems-and-proofs)). 
Could they be added as additional labels?

**Comment by jjallaire** [comment]
That is indeed an oversight! Thanks for pointing this out, will get this resolved soon.

**Reply by mchlbckr** [reply]
Thanks @jjallaire! Let me know if I can help with translations..

**Comment by jjallaire** [comment]
I'm just about to submit a bunch of PRs w/ best effort translations from Google Translate. I will ping the original authors of those translations for their feedback -- feel free to weigh in on any or all of them as you see fit!

**Comment by mcanouil** [comment]
Closing as completed, see <https://github.com/quarto-dev/quarto-cli/commit/2987932daf6dca06db1f1cf1bc390ce9c73bbdf3>.
